superset/assets/package.json

```diff
@@ -51,6 +51,7 @@
     "@data-ui/sparkline": "^0.0.54",
     "@data-ui/theme": "^0.0.62",
     "@data-ui/xy-chart": "^0.0.61",
+    "@superset-ui/core": "^0.0.3",
```

Comment:
noting that this version is not released and depends on [this superset-ui PR](https://github.com/apache-superset/superset-ui/pull/5) which adds a  `text` parse method to `SupersetClient`